controller.js

```diff
@@ -12,3 +12,9 @@ exports.getReviews = (request, response, next) => {
         .then((reviews) => response.status(200).send({ reviews }))
         .catch(next);
 };
+
+exports.getReviewComments = (request, response, next) => {
+    return fetchReviewComments(request.params.review_id)
+        .then((comments) => response.status(200).send({ comments }))

```

Comment:
HINT: to handle the empty comments you will need to do something different here ;)

Reply:
Hi, if it should still send an object containing an empty array if there are no comments, do I need to change this? Only reason I can see is if I need to send a different status code. If that's the case, is it a 204 (No content)?